Lombiq.Tests.UI/Docs/SecurityScanning.md

```diff
@@ -19,6 +19,23 @@ You can create detailed security scans of your app with [Zed Attack Proxy (ZAP)]
 - While ZAP is fully managed for you, Docker needs to be available and running to host the ZAP instance. On your development machine, you can install [Docker Desktop](https://www.docker.com/products/docker-desktop/).
 - The full scan of a website with even just 1-200 pages can take 5-10 minutes. So, be careful to fine-tune the ZAP configuration to make it suitable for your app.
 
+## Limitations
+
+On Windows-based GitHub runners the security tests always fail with the following error:
+
+> The `docker.exe pull softwaresecurityproject/zap-stable:2.14.0 --quiet` command failed with the output below.
+> no matching manifest for windows/amd64 10.0.20348 in the manifest list entries.
+
+This is because the Docker installation is configured to use Windows images, while the [ZAP docker image](https://hub.docker.com/r/softwaresecurityproject/zap-stable/tags) is only available for Linux. If you rely on our [Lombiq GitHub Actions](https://github.com/Lombiq/GitHub-Actions) then you can configure it like this to disable a test, in this case `SecurityScanningTests`:
+```
+  call-build-and-test-workflow-windows:
+    name: Build and Test
+    uses: Lombiq/GitHub-Actions/.github/workflows/build-and-test-orchard-core.yml@dev
+    with:
+      machine-types: '["windows-latest"]'
+      test-filter: "FullyQualifiedName!~SecurityScanningTests"
+```
```

Comment:
First sounded like BS but actually useful!

Reply:
Yeah, I don't even read the first sentence, jump to the "However" part directly.

Reply:
Asked the CR team: https://discord.com/channels/1134356397673414807/1134374140397236334/1196183468430864424

Reply:
It took ten million years to figure out that the discord link didn't work for me because I haven't joined the coderabbit-chat channel yet. I had to hunt down an invite link on coderabbit's web site first.  I don't even think this is a boomer moment on my end! I mean what UX genius thought that the error message should say "You find yourself in a strange place." instead of "You are not invited to this channel, idiot!" or something like that...

Reply:
Ten million years seems like an exaggeration to me, your argument is invalid.

Reply:
Nah, it was like in Triangle except my time looping axe murderer was bad UI design. 😁

Reply:
That should have been exhausting :D.